Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -93,6 +98,11 @@ protected static IMixedRealityInputSystem InputSystem
         /// </summary>
         public int Dimensions = 1;
 
+        /// <summary>
+        /// The Dimension value to set on start
+        /// </summary>
+        public int StartDimensionIndex = 0;
```

Comment:
note: it's probably too much for this PR, but per the [Coding Guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers) it would be nice to make these serialized and private, and then expose the fields via public properties.

Reply:
I've been thinking about this for a while now, I'll at least make the new stuff follow the guidelines.